README.md

```diff
@@ -1,64 +1,87 @@
+- See as well [Windows Installation](#Windows-Installation)
 
-If there are languages that you don't want to use, remove their configuration and notes from your `init.lua` after copy and pasting (for example, in the mason configuration).
-
-### Windows Installation
+### Configuration And Extension
 
-Installation may require installing build tools, and updating the run command for `telescope-fzf-native`
+* Inside of your fork, feel free to modify any file you like! It's your fork!
+* Then there are two primary configuration options available:
+  * Include the `lua/kickstart/plugins/*` files in your configuration.
+  * Add new configuration in `lua/custom/plugins/*` files, which will be auto sourced using `lazy.nvim`
 
-See `telescope-fzf-native` documention for [more details](https://github.com/nvim-telescope/telescope-fzf-native.nvim#installation)
+You can also merge updates/changes from the repo back into your fork, to keep up-to-date with any changes for the default configuration
 
-This requires:
+#### Example: Adding an autopairs plugin
 
-- Install CMake, and the Microsoft C++ Build Tools on Windows
+In the file: `lua/custom/plugins/autopairs.lua`, add:
 
 ```lua
-use {'nvim-telescope/telescope-fzf-native.nvim', run = 'cmake -S. -Bbuild -DCMAKE_BUILD_TYPE=Release && cmake --build build --config Release && cmake --install build --prefix build' }
+-- File: lua/custom/plugins/autopairs.lua
+
+return {
+  "windwp/nvim-autopairs",
+  config = function()
+    require("nvim-autopairs").setup {}
+  end,
+}
 ```
 
-### Configuration
-
-You could directly modify the `init.lua` file with your personal customizations. This option is the most straightforward, but if you update your config from this repo, you may need to reapply your changes.
 
-An alternative approach is to create a separate `custom.plugins` module to register your own plugins. In addition, you can handle further customizations in the `/after/plugin/` directory (see `:help load-plugins`). See the following examples for more information. Leveraging this technique should make upgrading to a newer version of this repo easier. 
+This will automatically install `nvim-autopairs` and enable it on startup. For more information, see documentation for [lazy.nvim](https://github.com/folke/lazy.nvim).
 
-#### Example `plugins.lua`
+#### Example: Adding a file tree plugin
 
-The following is an example of a `plugins.lua` module (located at `$HOME/.config/nvim/lua/custom/plugins.lua`) where you can register your own plugins. 
+In the file: `lua/custom/plugins/filetree.lua`, add:
 
 ```lua
-return function(use)
-  use({
-    "folke/which-key.nvim",
-      config = function()
-        require("which-key").setup({})
-      end
-  })
-end
+return {
+  "nvim-neo-tree/neo-tree.nvim",
+  branch = "v2.x",
+  dependencies = {
+    "nvim-lua/plenary.nvim",
+    "nvim-tree/nvim-web-devicons", -- not strictly required, but recommended
+    "MunifTanjim/nui.nvim",
```

Comment:
Again, totally fine as is, but lua dependencies don't have to be explicitly defined with lazy.

With lazy, when a plugin loads, then first all its dependencies will be loaded. so in this case those three plugins would always be loaded, wether they are used or not.

If you don't specify the explicit dependencies, they would be automatically loaded when doing `require("plenary")` for example.